src/commands/import-resolver.ts

```diff
@@ -4,7 +4,7 @@ import {SnowpackConfig} from '../config';
 import {findMatchingMountScript, ImportMap} from '../util';
 import srcFileExtensionMapping from './src-file-extension-mapping';
 const cwd = process.cwd();
-const URL_HAS_PROTOCOL_REGEX = /^\w:\/\./;
+const URL_HAS_PROTOCOL_REGEX = /^(\w+:)?\/\//;
```

Comment:
Just realized there was a bug in this RegEx. `\w` matches a word _character_, not a word. So `\w+` is needed.

I also removed the character trailing the `//` (the `.`). I think that’s OK, because this was only used in a `.test()` below? I needed to remove that to strip just the protocol, not the protocol + 1 trailing char below.